exporters/otlp/recordable_test.cc

```diff
@@ -0,0 +1,64 @@
+#include "exporters/otlp/recordable.h"
+
+#include <gtest/gtest.h>
+
+using namespace opentelemetry::exporter::otlp;
+
+TEST(Recordable, SetIds)
+{
+    Recordable rec;
+
+    const uint8_t trace_id_buff[16] = {0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1};
+    const uint8_t span_id_buff[8] = {0,0,0,0,0,0,0,2};
+    const uint8_t parent_span_id_buff[8] = {0,0,0,0,0,0,0,3};
+
+    opentelemetry::nostd::span<const uint8_t, 16>trace_id_span(trace_id_buff, 16);
+    opentelemetry::nostd::span<const uint8_t, 8> span_id_span(span_id_buff, 8);
+    opentelemetry::nostd::span<const uint8_t, 8> parent_span_id_span(parent_span_id_buff, 8);
+
+    opentelemetry::trace::TraceId trace_id(trace_id_span);
```

Comment:
Using buff and span helper variables is a bit overly verbose, I think it would work to just define TraceId as

const opentelemetry::trace::TraceId trace_id(
      std::array<const uint8_t, opentelemetry::trace::TraceId::kSize>(
          {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1}));

(same for other ids, and delete buff and span variables). It is also better not to use magic numbers like 8 and 16, but use trace::TraceId::kSize and 
trace::SpanId::kSize intead.

Reply:
Thanks! I do need the buffer later on for the comparison, and if I initialize this way I'll need to copy the TraceId representation back out into another buffer later. I've now removed the intermediary nostd::span but kept the buffer, please let me know what you think. I've also removed any magic numbers, although I'm not sure what would happen to my buffer initialization if the kSize changes.